src/modules/AutoinstSoftware.rb

```diff
@@ -706,7 +707,11 @@ def Export
       # set to true. This differs from the installation process. So we have
       # to set "install_recommended" to true in order to reflect the
       # installation process and cannot use the package bindings. (bnc#990494)
-      Ops.set(s, "install_recommended", true)
+      # OR: Each product (e.g. CASP) can set it in the control.xml file.
+      rec = ProductFeatures.GetFeature("software",
+        "clone_install_recommended_default")
+      # rec can be an empty String too if it has not been set in control.xml
+      s["install_recommended"] = rec.is_a?(FalseClass) ? false : true
```

Comment:
I'm not sure about this. `GetFeature` will return `"yes"`, `"no"` and the empty string `""`. So why do you use the `rec.is_a?(FalseClass)` condition?